connect/client.go

```diff
@@ -106,6 +109,53 @@ type restClient struct {
 	client    httpClient
 }
 
+func (rs *restClient) GetFiles(itemUUID string, vaultUUID string) ([]onepassword.File, error) {
```

Comment:
Can we move these functions down a bit so all file functions are grouped like they are in the interface declaration?

Reply:
Thanks for the tip! This should be resolved now.